build.sbt

```diff
@@ -53,7 +53,12 @@ lazy val root = Project(
   base = file(".")
 ).aggregate(aggregatedProjects: _*)
  .settings(rootSettings: _*)
- .settings(unidocRootIgnoreProjects := Seq(remoteTests, benchJmh, protobuf, akkaScalaNightly, docs))
+ .settings(unidocRootIgnoreProjects :=
+   (CrossVersion.partialVersion(scalaVersion.value) match {
+     case Some((2, n)) if n == 12 ⇒ Seq(remoteTests, benchJmh, protobuf, akkaScalaNightly, docs)
+     case _                       ⇒ aggregatedProjects // ignore all, only unidoc when scalaVersion is 2.12
```

Comment:
Probably a good idea, but if adding here it would always log when switching to 2.11 and not only when running unidoc with 2.11. I guess nobody is switching to 2.11 apart from the automatic release or some jenkins job.

I guess what we want to avoid is spending time on not understanding why unidoc is not running when we switch to 2.13 as the default (in a few years). I'll switch this to only use `aggregatedProjects` (ignore all) for 2.11 instead.